batch/batch/front_end/front_end.py

```diff
@@ -1626,10 +1629,52 @@ def plot_job_durations(container_statuses: dict, batch_id: int, job_id: int):
     return json.dumps(fig, cls=plotly.utils.PlotlyJSONEncoder)
 
 
+
+
 def plot_resource_usage(
         resource_usage: Optional[Dict[str, Optional[pd.DataFrame]]]
 ) -> Optional[str]:  # pylint: disable=unused-argument
-    return None
+    fig = make_subplots(
+        rows=2, cols=1,
+        subplot_titles=("Memory", "CPU Usage")
+    )
+    colors = {'input': 'red', 'main': 'green', 'output': 'blue'}
+
+    if resource_usage is None:
+        return None
+    for container_name, df in resource_usage.items():
+        if df is None:
+            continue
+
+        time = pd.to_datetime(df['time_msecs'], unit='ms')
+        mem = df['memory_in_bytes']
+        cpu = df['cpu_usage']
+        fig.add_trace(
+                    go.Scatter(
+                              x=time,
+                              y=cpu,
+                              legendgroup=container_name,
+                              name=container_name,
+                              mode="lines",
```

Comment:
nit: can we put `"lines"` in single quotes here and below?

Reply:
Added change